packages/contracts-bedrock/contracts/test/AttestationDisputeGame.t.sol

```diff
@@ -0,0 +1,245 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import "forge-std/Test.sol";
+
+import "../libraries/DisputeTypes.sol";
+import "../libraries/DisputeErrors.sol";
+
+import { L2OutputOracle } from "../L1/L2OutputOracle.sol";
+import { SystemConfig } from "../L1/SystemConfig.sol";
+
+import { AttestationDisputeGame } from "../dispute/AttestationDisputeGame.sol";
+import { IDisputeGameFactory } from "../dispute/IDisputeGameFactory.sol";
+import { IDisputeGame } from "../dispute/IDisputeGame.sol";
+import { IBondManager } from "../dispute/IBondManager.sol";
+import { BondManager } from "../dispute/BondManager.sol";
+import { DisputeGameFactory } from "../dispute/DisputeGameFactory.sol";
+import { Portal_Initializer } from "./CommonTest.t.sol";
+
+/**
+ * @title AttestationDisputeGame_Test
+ */
+contract AttestationDisputeGame_Test is Portal_Initializer {
+    bytes32 constant TYPE_HASH = 0x2676994b0652bcdf7968635d15b78aac9aaf797cc94c5adeb94376cc28f987d6;
+
+    L2OutputOracle l2oo;
+
+    DisputeGameFactory factory;
+    BondManager bm;
+    AttestationDisputeGame disputeGameImplementation;
+    AttestationDisputeGame disputeGameProxy;
+
+    // SystemConfig `attestorSet` keys
+    uint256[] attestorKeys;
+
+    // Emitted when a new dispute game is created by the [DisputeGameFactory]
+    event DisputeGameCreated(
+        address indexed disputeProxy,
+        GameType indexed gameType,
+        Claim indexed rootClaim
+    );
+
+    function setUp() public override {
+        super.setUp();
+
+        factory = new DisputeGameFactory(address(this));
+        vm.label(address(factory), "DisputeGameFactory");
+        bm = new BondManager(factory);
+        vm.label(address(bm), "BondManager");
+
+        vm.label(address(systemConfig), "SystemConfig");
+        // Transfer ownership of the system config to this contract
+        vm.prank(address(1));
+        systemConfig.transferOwnership(address(this));
+
+        // Add 5 signers to the attestor set
+        for (uint256 i = 1; i < 6; i++) {
+            attestorKeys.push(i);
+            systemConfig.setAttestor(vm.addr(i), true);
+        }
+        systemConfig.setAttestationThreshold(5);
+
+        // {
```

Comment:
dead comments?

Reply:
yep- port isn't done yet, @refcell's version used a mock L2OO / SystemConfig, in the process of using the real thing here. Will pull it out of draft when this PR is good to go 💯